### DISCUSSION TITLE
How to recreate auth.users table

### DISCUSSION THREAD

**Discussion by unhyif** [supabase/supabase#17830] [Questions] [answered]
Hi there, is there a way to recreate the default `auth.users` table after having deleted it? I deleted it due to my poor skills at migration. 🥲
I can't sign up at all now and I assume this is because the `auth.users` table has gone. Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
I've not seen anyone post an easy way.  Is this a pro instance, you could use a backup.

You could create a new instance and either copy auth.users from there, or copy your data over to it (safer) with psql and the direct database connections.  Note the instances have to be very near in time because auth.users does change, and there is still risk a new version deploys between your old instance and new.

You could also see if support can restore it, especially if you are pro.

**Reply by unhyif** [reply]
@GaryAustin1 Thank you for your answer. Unfortunately I'm using free tier. I was curious if supabase is providing a certain SQL query that can automatically create default `auth.users` table but it sounds like no. 😅 Have a nice day!

**Comment by GeKaixing** [comment]
Restart the email registration
